src/utils.js

```diff
@@ -61,3 +61,9 @@ export function parseVersion(version) {
 
     return version
 }
+
+// Kong 1.x changes these references from plugin.${entity}_id to
+// plugin.${entity}.id. This function enables us to support both formats.
+export function getForeignEntityID(primary, entityType) {
+    return primary[`${entityType}_id`] || (primary[entityType] && primary[entityType].id)
+}
```

Comment:
Where does the word "foreign" come from? Is this language that is used in the kong documentation? Would "associated" or "related" communicate the same thing without implying that it's an entity that exists outside kong?

Reply:
No, this is entirely my own poor word choice. I'll go with associated!

Reply:
It's certainly not a poor word choice just because it threw me. I'm curious what @jasalisbury and @mjlumetta think since y'all will probably be working with the code more than I will be in the future.

Reply:
I can go either way, I don't read "foreign" as being outside of kong, I read it more in the DB "foreign key" sense. If you do end up renaming it @kalilsn I think you missed renaming some invocations up in `kongState.js`

Reply:
That was the kind of "foreign" that I was thinking of, mostly because they actually are foreign keys in the underlying postgres db. I don't think that's a good reason to name them that way, since the admin API should be a higher level abstraction than the db. I had a lot of trouble naming this, and I think "associated" is much clearer.

>you missed renaming some invocations up in kongState.js

Thanks! Turns out you have to actually save changes if you want to persist them!